<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>당신 곁의 잔여백신_백발백신</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../AppData/Roaming/Microsoft/Windows/Network Shortcuts/testResource.qrc">
    <normaloff>:/background/로고_아이콘.png</normaloff>:/background/로고_아이콘.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="maintxt_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;데이터 수집을 통해&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="maintxt_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;확인해보세요&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../AppData/Roaming/Microsoft/Windows/Network Shortcuts/testResource.qrc">:/background/로고_연한거 (1).png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>데이터 수집하기</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="maintxt_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;우리동네 잔여백신 발생 현황&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>411</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../AppData/Roaming/Microsoft/Windows/Network Shortcuts/testResource.qrc">:/background/bg.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bg_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../AppData/Roaming/Microsoft/Windows/Network Shortcuts/testResource.qrc">:/background/bg_2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bg_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>501</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../AppData/Roaming/Microsoft/Windows/Network Shortcuts/testResource.qrc">:/background/bg-Light Gray.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>사용 전 필독 사항</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="maintxt_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;신청 성공 확률이 높은 병원과 시간대를&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>수집 결과 확인 </string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>bg_1</zorder>
   <zorder>bg_2</zorder>
   <zorder>maintxt_2</zorder>
   <zorder>maintxt_4</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>maintxt_1</zorder>
   <zorder>bg_3</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>maintxt_3</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>logo</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../AppData/Roaming/Microsoft/Windows/Network Shortcuts/testResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>must()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>collect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>must()</slot>
  <slot>collect()</slot>
 </slots>
</ui>
